cli/src/pcluster/resources/imagebuilder/parallelcluster_validate.yaml

```diff
@@ -0,0 +1,454 @@
+name: ParallelClusterValidate
+description: Validate ParallelCluster AMI
+schemaVersion: 1.0
+
+phases:
+  - name: validate
+    steps:
+      ### basic ###
+      # Exit code for failures
+      - name: Fail
+        action: ExecuteBash
+        inputs:
+          commands:
+            - echo 1
+
+      # Check input base AMI OS and get OS information, the output should be like centos.7 | centos.8 | amzn.2 | ubuntu.18.04 | ubuntu.20.04
+      - name: OperatingSystemRelease
```

Comment:
Since this component must be paired with the main pcluster component, is it needed to redefines all this utility actions in here?
Cannot we just reference the utilities from the `build` phases, e.g. `{{ build.OperatingSystemRelease.outputs.stdout }}`?

Reply:
we can't reuse stage output in other component